routing-daemon/lib/openshift/routing/controllers/asynchronous.rb

```diff
@@ -328,57 +322,13 @@ def delete_pool pool_name
       # will block on the :create_pool event that is blocking on the
       # previous :delete_pool event.
       #
-      # Along similar lines, checking for :delete_route and
-      # :delete_pool_member is sufficient; it is not necessary to check
-      # for :create_route and :add_pool_member.
-      #
```

Comment:
We could keep the comment and simply s/`:delete_route`/`:delete_pool_alias`/ and s/`:create_route`/`:add_pool_alias`/.